content/200-concepts/100-components/01-prisma-schema/06-relations.mdx

```diff
@@ -1088,6 +1089,108 @@ model Post {
+
+The back relation fields are missing from the `Post` to `PostCategories` and `Category` to `PostCategories` models.
+
+```prisma
+model Post {
+  id            Int         @id @default(autoincrement())
+  title         String
+  categories    Category[]
+}
+
+model PostCategories {
+  postId        User        @relation(fields: [post], references: [id])
+  post          Post
+  categoryId    Facility    @relation(fields: [category], references: [id])
+  category      Category
+  @@id([post, category])
+}
+
+model Category {
+  id             Int        @id @default(autoincrement())
+  name           String
+  posts          Post[]
+}
+```
+
+To fix this the `Post` model needs to have a many relation field defined with the relation table `PostCategories`. The same applies to the `Category` model.
+
+This is because the relation model forms a 1-to-many relationship with the other two models its joining.
+
+```prisma highlight=5,20;add|4,19;delete
+model Post {
+  id              Int               @id @default(autoincrement())
+  title           String
+  categories      Category[]
+  postCategories  PostCategories[]
+}
+
+model PostCategories {
+  postId        User                @relation(fields: [post], references: [id])
+  post          Post
+  categoryId    Facility            @relation(fields: [category], references: [id])
+  category      Category
+  @@id([post, category])
+}
+
+model Category {
+  id              Int                @id @default(autoincrement())
+  name            String
+  posts           Post[]
+  postCategories  PostCategories[]
+}
+```
+
+### Using the <inlinecode>@relation</inlinecode> attribute with a many-to-many relationship
+
+It might seem logical to add a `@relation("Post")` annotation to a relation field on your model when composing an implicit many-to-many relationship.
+
+```prisma
+model Post {
+  id            Int         @id @default(autoincrement())
+  title         String
+  categories    Category[]  @relation("Category")
+}
+
+model Category {
+  id             Int      @id @default(autoincrement())
+  name           String
+  posts          Post[]   @relation("Post")
+}
+```
+
+This however tells Prisma to expect **two** separate many-to-many relationships. See [disambiguating relations](#disambiguating-relations) for more information on using the `@relation` attribute.
+
+The following example is the correct way to define an implicit many-to-many relationship.
+
+```prisma highlight=4,11;delete|5,12;add
+model Post {
+  id            Int         @id @default(autoincrement())
+  title         String
+  categories    Category[]  @relation("Category")
+  categories    Category[]
+}
+
+model Category {
+  id             Int      @id @default(autoincrement())
+  name           String
+  posts          Post[]   @relation("Post")
+  posts          Post[]
+}
+```
+
```

Comment:
You could show a second alternative here that uses the same relation name:

```suggestion
In case you do want to control the name of the underlying relation table, you can do this as follows:

```prisma
model Post {
  id            Int         @id @default(autoincrement())
  title         String
  categories    Category[]  @relation("CategoryPostRelation")
}
model Category {
  id             Int      @id @default(autoincrement())
  name           String
  posts          Post[]   @relation("CategoryPostRelation")
}
```
```

Reply:
Yep! This is what i meant in slack! 😁